concepticondata/conceptlists/Lundgren-2020-275.tsv

```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-165	165	165	MIX (TR.)~PRESS		
+Lundgren-2020-275-166	166	166	MONKEY	1350	MONKEY
+Lundgren-2020-275-167	167	167	MOON	1313	MOON
+Lundgren-2020-275-168	168	168	MOSQUITO	1509	MOSQUITO
+Lundgren-2020-275-169	169	169	MOURN	1955	GRIEVE
+Lundgren-2020-275-170	170	170	MOUSE~RAT	2139	MUROID (MOUSE OR RAT)
+Lundgren-2020-275-171	171	171	MOUTH	674	MOUTH
+Lundgren-2020-275-172	172	172	NAIL (BODY PART)	1896	FINGERNAIL OR TOENAIL
+Lundgren-2020-275-173	173	173	NAME		
+Lundgren-2020-275-174	174	174	NECK~THROAT		
+Lundgren-2020-275-175	175	175	NEPHEW	1063	NEPHEW
+Lundgren-2020-275-176	176	176	NEW	1231	NEW
+Lundgren-2020-275-177	177	177	NIGHT	1233	NIGHT
+Lundgren-2020-275-178	178	178	NOSE	1221	NOSE
+Lundgren-2020-275-179	179	179	OBEY (TR.)	1635	OBEY
+Lundgren-2020-275-180	180	180	OPEN (TR.)	1156	OPEN
+Lundgren-2020-275-181	181	181	OTHER	197	OTHER
+Lundgren-2020-275-182	182	182	OWNER	1036	OWNER
+Lundgren-2020-275-183	183	183	PACA	316	PACA
+Lundgren-2020-275-184	184	184	PADDLE (N.)	234	PADDLE
+Lundgren-2020-275-185	185	185	PAINT~WRITE		
+Lundgren-2020-275-186	186	186	PATH	2252	PATH
+Lundgren-2020-275-187	187	187	PATIO		
+Lundgren-2020-275-188	188	188	PEANUT	2364	PEANUT
+Lundgren-2020-275-189	189	189	PERSON	683	PERSON
+Lundgren-2020-275-190	190	190	PET		
+Lundgren-2020-275-191	191	191	PINEAPPLE	2440	PINEAPPLE
+Lundgren-2020-275-192	192	192	PIRANHA	317	PIRANHA
+Lundgren-2020-275-193	193	193	PIT VIPER SP.		
+Lundgren-2020-275-194	194	194	PLANT (TR.)	1486	PLANT (SOMETHING)
+Lundgren-2020-275-195	195	195	POOR	1674	POOR
+Lundgren-2020-275-196	196	196	PREY		
+Lundgren-2020-275-197	197	197	RAIN	2108	RAINING OR RAIN
+Lundgren-2020-275-198	198	198	RED	156	RED
+Lundgren-2020-275-199	199	199	REST (INTR.)	168	REST
+Lundgren-2020-275-200	200	200	RETURN (INTR.)	581	COME BACK
+Lundgren-2020-275-201	201	201	RIVER	666	RIVER
+Lundgren-2020-275-202	202	202	ROAST (TR.)	1962	ROAST (SOMETHING)
+Lundgren-2020-275-203	203	203	ROCK~SHAKE		
+Lundgren-2020-275-204	204	204	ROPE~THREAD		
+Lundgren-2020-275-205	205	205	ROUND	1395	ROUND
+Lundgren-2020-275-206	206	206	SANDFLY	95	SANDFLY
+Lundgren-2020-275-207	207	207	SAP~CAUIM		
+Lundgren-2020-275-208	208	208	SCRAPE (TR.)	1767	SCRAPE
+Lundgren-2020-275-209	209	209	SHAMAN	394	SHAMAN (FOLK HEALER)
+Lundgren-2020-275-210	210	210	SHINE (INTR.)	708	SHINE
+Lundgren-2020-275-211	211	211	SHOUT	715	SHOUT
+Lundgren-2020-275-212	212	212	SING (INTR.)	1261	SING
+Lundgren-2020-275-213	213	213	SISTER	1263	SISTER
+Lundgren-2020-275-214	214	214	SISTER-IN-LAW (F. EGO)		
+Lundgren-2020-275-215	215	215	SIT (INTR.)	1416	SIT
+Lundgren-2020-275-216	216	216	SKIN	763	SKIN
+Lundgren-2020-275-217	217	217	SLOTH	2446	SLOTH
+Lundgren-2020-275-218	218	218	SMELL (INTR. TR.)	2124	SMELL
+Lundgren-2020-275-219	219	219	SMOKE (N.)	778	SMOKE (EXHAUST)
+Lundgren-2020-275-220	220	220	SMOKE FOOD		
+Lundgren-2020-275-221	221	221	SMOOTH	1234	SMOOTH
+Lundgren-2020-275-222	222	222	SNAKE	730	SNAKE
+Lundgren-2020-275-223	223	223	SON (M. EGO)		
```

Comment:
This is an important concept that we should add. It is SON (OF MALE), so it is a kinship term, similar to the ones we have in "SISTER (OF MALE PERSON)". They use "ego" to denote "person". So I suggest to add DAUGHTER and SON (OF MALE EGO) to concepticon (and on the long run, we should switch our nomenclature on kinship terms also. 

This requires to add a relation between SON which is broader than SON (OF MALE EGO) etc.

Reply:
Agreed. What about other kinship terms in relation to SISTER-IN-LAW (F. EGO) and GRANDCHILD (F. EGO)? Shall I add the corresponding concepts for them too?